src/MyHealth/MyHealthContent.spec.tsx

```diff
@@ -1,61 +1,73 @@
 import React from "react"
-import { fireEvent, render } from "@testing-library/react-native"
+import { fireEvent, render, waitFor } from "@testing-library/react-native"
 import { useNavigation } from "@react-navigation/native"
 
 import { SymptomLogContext } from "./SymptomLogContext"
-import { SymptomLogEntry, CheckInStatus } from "./symptoms"
+import { SymptomLogEntry } from "./symptoms"
 
-import OverTime from "./OverTime"
+import MyHealthContent from "./MyHealthContent"
 import { factories } from "../factories"
 import { posixToDayjs } from "../utils/dateTime"
-import { MyHealthStackScreens } from "../navigation"
+import { ModalStackScreens, MyHealthStackScreens, Stacks } from "../navigation"
 
 jest.mock("@react-navigation/native")
 
-describe("OverTime", () => {
-  describe("when the user has no checkins or symptom logs", () => {
-    it("displays a 'no logs' message", () => {
-      const { getByText } = render(
+describe("MyHealthContent", () => {
+  it("allows the user to add a symptom log entry", async () => {
+    const navigateSpy = jest.fn()
+    ;(useNavigation as jest.Mock).mockReturnValue({
+      navigate: navigateSpy,
+    })
+    const { getByLabelText } = render(
+      <SymptomLogContext.Provider value={factories.symptomLogContext.build({})}>
```

Comment:
```suggestion
      const initialContext = factories.symptomLogContext.build()
      <SymptomLogContext.Provider value={initialContext}>
```

I try to avoid inlining things like this as they tend to grow unruly over time.